fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -250,6 +252,13 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     @VariableMgr.VarAttr(name = QUERY_TIMEOUT)
     private int queryTimeoutS = 300;
 
+    // Execution of a query contains two phase.
+    // 1. Deliver all the fragment instances to BEs.
+    // 2. Pull data from BEs, after all the fragments are prepared and ready to execute in BEs.
+    // queryDeliveryTimeoutS is the timeout of the first phase.
+    @VariableMgr.VarAttr(name = QUERY_DELIVERY_TIMEOUT)
+    private int queryDeliveryTimeoutS = 300;
```

Comment:
I'm afraid `S` does not obey the java naming rule....

Reply:
It is to be consistent with `queryTimeoutS`, and also obeys lower camel case.